pandas/core/frame.py

```diff
@@ -1598,8 +1597,9 @@ def to_feather(self, fname):
         from pandas.io.feather_format import to_feather
         to_feather(self, fname)
 
-    @Substitution(header='Write out column names. If a list of string is given, \
-it is assumed to be aliases for the column names')
+    @Substitution(header='Write out the column names. If a list of strings '
+                         'is given, it is assumed to be aliases for the '
+                         'column names')
```

Comment:
I suggest moving on and leaving this be.  As it is `to_csv.__doc__` is all in one piece and I'm not inclined to disturb it.

Unrelated: is there a time in the foreseeable future when `Panel` etc will be deprecated-as-in-deleted?